WebPhapp/client/src/App/pages/PrescriptionAdd.js

```diff
@@ -69,55 +69,141 @@ class PrescriptionAdd extends Component {
 
     return (
       <div className="form-group">
-        <h1>Add Prescription </h1>
-        <form>
+          <nav className="navbar navbar-horizontal navbar-expand-lg navbar-dark bg-primary">
```

Comment:
You should remove this navbar... This is included in the header.js file.

Reply:
👍